soroban-sdk/src/env.rs

```diff
@@ -244,6 +244,28 @@ impl Env {
     pub fn log_value<V: IntoVal<Env, RawVal>>(&self, v: V) {
         internal::Env::log_value(self, v.into_val(self));
     }
+
+    #[doc(hidden)]
```

Comment:
Can doc comments be added for each function and the `#[doc(hidden)]` removed? `#[doc(hidden)]` is only necessary on functions we don't plan on having developers use.